### PR TITLE
refer interested readers to the roslyn repo feature status

### PR DESCRIPTION
**Pull request by BillWagner**
The C# team maintains the referenced page with feature status for upcming releases. Readers interested in "What's New" are likely also interested in "What's coming soon."

/cc @jaredpar

### CODE REVIEW

**Review by jaredpar — APPROVED** [review]

File: docs/csharp/whats-new/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ ms.assetid: 77deec51-a14d-46d4-9bb3-faf449477149
 
 This page provides a roadmap of new features in each major release of
 the C# language. The following links provide detailed information on the
-major features added in each release.
+major features added in each release. This section provides information on new feature that have been released, either in a general release, or in a public preview. Detailed language feature status, including features considered for upcoming releases can be found [on the roslyn repository](https://github.com/dotnet/roslyn/blob/master/docs/Language%20Feature%20Status.md) on GitHub.
```
Comment (mairaw) [inline docs/csharp/whats-new/index.md]: on new feature or new features?

dotnet/roslyn repository? we usually put org/reponame unless you think it's not needed for roslyn

**Review by mairaw — APPROVED** [review]
LGTM. Left a few comments for you to consider.

**Review by Thraka — APPROVED** [review]
LGTM but a thought on the intro paragraph. It seems to flow strange to me with 3 different introductory clauses:

This page provides ...
The following links provide ... 
This section provides ... 

Perhaps there is a better way to state all this?

**Comment by BillWagner** [comment]
Closing manually. The GitHub status issues meant that this did not close when it was merged.

**Pull request merged**